x-pack/plugin/sql/qa/src/main/resources/docs.csv-spec

```diff
@@ -713,8 +713,8 @@ SELECT SCORE(), * FROM library WHERE MATCH(name, 'dune') ORDER BY SCORE() DESC;
 
     SCORE()    |    author     |       name        |  page_count   |    release_date    
 ---------------+---------------+-------------------+---------------+--------------------
-2.288635       |Frank Herbert  |Dune               |604            |1965-06-01T00:00:00Z
-1.8893257      |Frank Herbert  |Dune Messiah       |331            |1969-10-15T00:00:00Z
+1.0402887       |Frank Herbert  |Dune               |604            |1965-06-01T00:00:00Z
+0.8587844      |Frank Herbert  |Dune Messiah       |331            |1969-10-15T00:00:00Z
 1.6086555      |Frank Herbert  |Children of Dune   |408            |1976-04-21T00:00:00Z
```

Comment:
This doesn't look right?  All the scores should change, and currently things aren't in score order

Reply:
ok I will double check if I fixed all the issues in tests, and if so I will check why this is the case here.

Reply:
Let me add that I have just indexed the same documents used for these tests and verified manually that all of the 4 documents matching this query get an updated score with the bm25 change. Seems like something weird happens with these tests, not sure what exactly.

Reply:
A mixed cluster with nodes in 6x and 7 ?

Reply:
possibly, yet I would expect these tests to run both in multi-node and single-node. which makes me expect that I should not get a green build with the current changes. Single-node should require to update all the scores. I may not be getting how test tests are run.

Reply:
@javanna indeed, there is something wrong here. The tests pass and, looking at the logs it produces, it seems like it's not comparing the output from the `csv-spec` with the actual output of the query being executed... am looking into it.

Reply:
@javanna the issue is [here](https://github.com/elastic/elasticsearch/blob/6fbc166afd75776363ebba897fcf032ce8a9d999/x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/JdbcAssert.java#L223). An assertion with 1.0 delta which, in the case of SCORE(), is quite relevant and a deal breaker.